src/pages/Home/Home.jsx

```diff
@@ -0,0 +1,53 @@
+import HomeStyles from "./Home.module.css";
+import { useVideos } from "../../context/VideosContest";
+import { BottomNav } from "../../components/BottomNav/BottomNav";
+import { GenreSlider } from "../../components/GenreSlider/GenreSlider";
```

Comment:
You can use absolute imports it eliminates all the ../

Reply:
okeiy will try later,actually i tried in last project but then i got some problem in all of the imports like and getting errors so i reverted the imports.
will read about it and then implement.
thanks for review!